nixos/modules/services/misc/gitlab.nix

```diff
@@ -613,7 +778,8 @@ in {
     };
 
     systemd.services.gitlab = {
-      after = [ "gitlab-workhorse.service" "gitaly.service" "network.target" "postgresql.service" "redis.service" ];
+      after = [ "gitlab-workhorse.service" "gitaly.service" "network.target" "postgresql.service" "redis.service" "keys.target" ];
+      wants = [ "keys.target" ];
```

Comment:
FYI https://github.com/NixOS/nixpkgs/pull/67394

Reply:
Good to know. Thanks!